lib/game.rb

```diff
@@ -0,0 +1,70 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]]
+
+  def initialize(p_1 = Human.new("X"), p_2 = Human.new("O"), board = Board.new)
+    @player_1 = p_1
+    @player_2 = p_2
+    @board = board
+  end
+
+  def current_player
+    self.board.turn_count % 2 == 0 ? player_1 : player_2
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def won? 
+    WIN_COMBINATIONS.detect do |win_array|
+############ Try to insert PROC?? ###########
```

Comment:
Not sure what this comment is about but your code is great. Good use of `detect` and `all`